ProjFS.Mac/PrjFSKext/ProviderMessaging.cpp

```diff
@@ -124,6 +124,7 @@ void ProviderMessaging_AbortOutstandingEventsForProvider(VirtualizationRootHandl
     Mutex_Release(s_outstandingMessagesMutex);
 }
 
+#ifndef KEXT_UNIT_TESTING
```

Comment:
Thats a better option, updating.
Previously since the messaging was in KAuthHandler we couldn't pull it apart.  Another advanage to the refactoring @pmj did here.